unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -379,6 +379,24 @@ class LowMachKernelHex8Mesh : public TestKernelHex8Mesh
 
   virtual ~LowMachKernelHex8Mesh() {}
 
+  void fill_mesh(const std::string& meshSpec = "generated:20x20x20")
+  {
+    const double one = 1.0;
+    const double two = 2.0;
+    const double oneVecTwelve[12] = {one, one, one, one, one, one, one, one, one, one, one, one};
+    unit_test_utils::fill_hex8_mesh(meshSpec, bulk_);
+    stk::mesh::create_edges(bulk_, meta_.universal_part());
+    partVec_ = {meta_.get_part("block_1")};
+
+    coordinates_ = static_cast<const VectorFieldType*>(meta_.coordinate_field());
+
+    EXPECT_TRUE(coordinates_ != nullptr);
+
+    stk::mesh::field_fill(two, *dnvField_);
```

Comment:
`dual_nodal_volume` is already part of the base class from which `LowMach*` is derived.